src/app/views/teams/team-create/CreateTeam.test.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { mount } from "enzyme";
+import CreateTeam from "./CreateTeam";
+import { WrapperComponent } from "../../../../tests/test-utils";
+import renderer from "react-test-renderer";
+
+let dispatchedActions = [];
+const mockDispatch = event => {
+    dispatchedActions.push(event);
+};
+
+const mockedAllUsers = {
+    count: 45,
+    users: [
+        {
+            forename: "Test",
+            lastname: "user",
+            email: "test@test.com",
+            groups: [],
+            status: "CONFIRMED",
+            active: true,
+            id: "aaa-bbb-ccc",
+            status_notes: "",
+        },
+        {
+            forename: "Test",
+            lastname: "user 2",
+            email: "test2@test.com",
+            groups: [],
+            status: "CONFIRMED",
+            active: true,
+            id: "ddd-eee-fff",
+            status_notes: "",
+        },
+        {
+            forename: "Test",
+            lastname: "user 3",
+            email: "test3@test.com",
+            groups: [],
+            status: "CONFIRMED",
+            active: true,
+            id: "ggg-hhh-iii",
+            status_notes: "",
+        },
+    ],
+};
+
+jest.mock("../../../utilities/api-clients/user", () => {
+    return {
+        getAll: jest.fn(() => {
+            return Promise.resolve(mockedAllUsers);
+        }),
+    };
+});
+describe("CreateTeam", () => {
+    const defaultBaseProps = {
+        dispatch: mockDispatch,
+        router: {
+            listenBefore: () => {},
+            setRouteLeaveHook: () => {},
+        },
+    };
+    describe("Given valid props on initial load", () => {
+        const component = mount(
+            <WrapperComponent>
+                <CreateTeam {...defaultBaseProps} />
+            </WrapperComponent>
+        );
+        it("renders the relevant components to screen", () => {
```

Comment:
this test seems to be exactly the same what snapshot does?

Reply:
it would be nice to actually test the functionality here in this component. the name suggests `create team` can you add tests for events?